android/jni/tun2socks/tun2socks.c

```diff
@@ -294,8 +297,13 @@ static void udp_fd_handler(UdpPcb* udp_pcb, int event) {
         return;
     }
 
+    struct socks_udp_header* header = (struct socks_udp_header*)udp_pcb->udp_recv_buffer;
```

Comment:
I added comments for now. This code will be refactored and put it its own header file once we design a full UDP solution.